EPS/0.2.1.1/linuxamd64.Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.8.1-slim-buster
 
-ENV EPS_VERSION 0.2.0
+ENV EPS_VERSION 0.2.1.1
 ENV EPS_SHA256 e0931428a5a01801528cd8d05a0fdc5ef708b33d491ebb6721edb9237d30c285
```

Comment:
different version but same hash?

Reply:
Sorry always for stupid Qs.  How to get from here: https://github.com/chris-belcher/electrum-personal-server/releases/tag/eps-v0.2.1.1 the expected hash?